src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -327,7 +327,7 @@ void CollectError(Type extensionType, Version minimumVersion, ExtensionStartupTy
                 }
             }
 
-            if (errors.Count > 0)
+            if (errors != null && errors.Count > 0)
```

Comment:
We can create a IsNullOrEmpty utility method for list and arrays. We are checking this a few times in this PR. Also, using `is null` and `is not null`

Reply:
We are checking that combination only here, in this PR. The other instance (line 162) is an or condition (not and).